workers/ohsome_quality_analyst/oqt.py

```diff
@@ -3,20 +3,115 @@
 Functions are triggert by the CLI and API.
 """
 import logging
-from typing import Optional
+from typing import Optional, Union
 
 from asyncpg.exceptions import UndefinedTableError
-from geojson import Feature
+from geojson import Feature, FeatureCollection, MultiPolygon, Polygon
 
 import ohsome_quality_analyst.geodatabase.client as db_client
 from ohsome_quality_analyst.base.indicator import BaseIndicator
 from ohsome_quality_analyst.base.report import BaseReport
-from ohsome_quality_analyst.utils.definitions import INDICATOR_LAYER
-from ohsome_quality_analyst.utils.helper import name_to_class
+from ohsome_quality_analyst.utils.definitions import GEOM_SIZE_LIMIT, INDICATOR_LAYER
+from ohsome_quality_analyst.utils.helper import loads_geojson, name_to_class
+
+
+async def create_indicator_as_geojson(
```

Comment:
Currently this function is only tested indirectly but completly through API and CLI tests.

Not sure if we need additional dedictated tests. If so I would like to change the test structure in a way to make it easy to cover all possible input parameters for "compute indicator/report" functions in `oqt.py`, CLI and API. They all have very similar input parameter.

Reply:
Ideally the function would be tested explicitly in unittests, but you want to change something fundamental first, you can change that in another PR.

Reply:
I added to issues for this case:
https://github.com/GIScience/ohsome-quality-analyst/issues/114
https://github.com/GIScience/ohsome-quality-analyst/issues/115